### DISCUSSION TITLE
Consider changing Indent Blankline highlight context color

### DISCUSSION THREAD

**Discussion by CleoMenezesJr** [NvChad/NvChad#2393] [Ideas]
Indent Blankline context highlight seems hasn't a different color than the selection color in visual mode. This happens in all color schemes.
Visually, it is (almost?) impossible to know what I am selecting.

[Screencast from 2023-09-16 22-18-14.webm](https://github.com/NvChad/NvChad/assets/54215258/82e886ec-05a9-4adb-9e68-2af8ef93c433)

_If you don't understand what I'm doing in the video. I show it repeatedly by selecting below and then above where the Indent Blankline context highlight is._

**Comment by siduck** [comment]
@CleoMenezesJr override its hl group, pls read our docs

**Reply by CleoMenezesJr** [reply]
This was my first idea to fix this on my end. I was just commenting because I think it would be interesting for it to be upstream.

**Reply by siduck** [reply]
@CleoMenezesJr https://nvchad.com/docs/config/theming

**Reply by halturin** [reply]
i've read it but didn't get how to solve this issue, what magic word must be used to change the selection color and what these changes could affect too :)

**Reply by siduck** [reply]
@halturin  check base46's blankline integration

**Reply by AntonKrychevskyy** [reply]
@siduck Hi. I am stumbling all day. Initially I wanted to set dimmed colored indentation lines which supposed to change to brighter color when highlighting context in focus. 

I've added highlight group list in plugin configration. But for some reason adding corresponding colors to theme polish_hl or to highlights.lua has absolutely no effect. Finally I've managed to add colors next way `vim.cmd [[highlight IndentBlanklineIndent1 guifg=#7f505d gui=nocombine]]`. 

But IndentBlanklineChar, IndentBlanklineSpaceChar, IndentBlanklineContextChar and IndentBlanklineContextStart don't change their color in any way.

**Reply by siduck** [reply]
@AntonKrychevskyy  nvchad v2.0 uses blankline old version, v3.0 uses the new version which has easy config & tweaking, i'll update this weekend and let you know.
